package.json

```diff
@@ -64,5 +64,8 @@
       "/node_modules/",
       "/test/package/"
     ]
+  },
+  "dependencies": {
+    "traverse": "^0.6.6"
```

Comment:
I'm not a huge fan of adding a dependency. Is this possible without (and without having to write a lot of custom code)?

Reply:
` Is this possible without (and without having to write a lot of custom code)?`
Sure. What we actually need is a pretty simple traverse algorithim. I can write and embed it if you agree with the general idea of this PR.

Reply:
Currently this package has no dependencies and I want to keep it that way. If you could write a very simple custom solution which does not add a lot of complexity, only for this specific use-case, it would be great!